kaldb/src/test/java/com/slack/kaldb/logstore/schema/ConvertFieldValueAndDuplicateTest.java

```diff
@@ -102,8 +102,9 @@ public void testListTypeInDocumentWithoutFullTextSearch() throws IOException {
         build(CONVERT_VALUE_AND_DUPLICATE_FIELD, false, meterRegistry);
     assertThat(docBuilder.getIndexFieldConflictPolicy())
         .isEqualTo(CONVERT_VALUE_AND_DUPLICATE_FIELD);
-    assertThat(docBuilder.getSchema().size()).isEqualTo(17);
-    assertThat(docBuilder.getSchema().keySet()).contains(LogMessage.SystemField.ALL.fieldName);
+    assertThat(docBuilder.getSchema().size()).isEqualTo(16);
+    assertThat(docBuilder.getSchema().keySet())
```

Comment:
two lines above where we create `SchemaAwareLogDocumentBuilderImpl` we pass false for enableFullTextSearch